recipe/meta.yaml

```
@@ -43,8 +43,7 @@ requirements:
 
 test:
   commands:
```

Might as well enable the optional tests:

```suggestion
  requires:
    - r-bit64
    - r-r.utils
  commands:
```

---

ok works for me! I wasn't sure if maintainers here would be open to adding those extra tests and dependencies.